smart_campus/app/views.py

```diff
@@ -2,14 +2,21 @@
 from django.views.decorators.csrf import csrf_exempt
 from django.views.decorators.http import require_POST
 from django.contrib import auth
-from django.http import HttpResponseRedirect, JsonResponse
+from django.http import HttpResponseRedirect, JsonResponse, HttpResponseForbidden
+from django.shortcuts import get_object_or_404
 from django.contrib.auth.decorators import login_required
 from django.contrib import messages
+from django.contrib.gis.geos import Point
+from django.conf import settings
+
+import os
```

Comment:
I suggest that you can follow the import sequence of pep8.
i.e.`std libs` -> `3rd party libs` (including Django) -> `local packages`
But it's ok that you import Django first in a Django project.
Just note that you should follow the same rule across the project.